core/embed/rust/src/ui/lerp.rs

```diff
@@ -66,31 +66,27 @@ macro_rules! impl_lerp_for_unsigned {
 }
 
 impl_lerp_for_signed!(i16);
-impl_lerp_for_signed!(i32);
 impl_lerp_for_unsigned!(u8);
 impl_lerp_for_unsigned!(u16);
-impl_lerp_for_unsigned!(u32);
-
-impl_lerp_for_signed!(f32);
```

Comment:
any specific reason to remove those? they won't be compiled in when unused, and it seems generally useful to have them available for all types

Reply:
You're right - I will revert this change.

Reply:
fb6b11b